src/targets/crates.ts

```diff
@@ -196,12 +214,16 @@ export class CratesTarget extends BaseTarget {
    * @returns A promise that resolves when the upload has completed
    */
   public async publishPackage(crate: CratePackage): Promise<any> {
-    const args = [
-      'publish',
+    const args = this.config.noDevDeps
+      ? ['hack', 'publish', '--allow-dirty', '--no-dev-deps']
```

Comment:
Can we also install `cargo-hack` in Dockerfile, please?

Reply:
Excellent suggestion, added!